### DISCUSSION TITLE
Copilot.lua: auth not running

### DISCUSSION THREAD

**Discussion by 0xccx** [LazyVim/LazyVim#832] [Q&A] [answered]
After enabling Copilot I am not able to run `:Copilot auth`

I get the following error instead

`Client 1 quit with exit code 1 and signal 0`

Similarly, I get hundreds of the above errors while in LazyVim's config screen.

Any ideas how to troubleshoot? Many thanks

**Comment by folke** [comment]
I have no idea. Maybe you don't have `nodejs` installed? Did you already check `:checkhealth`?

**Reply by 0xccx** [reply]
I **didn't** have nodejs installed prior to installing LazyVim but I installed it after. I also installed some missing LuaJIT libraries that were missing and `:checkhealth` currently flags no errors

**Reply by folke** [reply]
Missing luajit libraries? That should not be needed, since that is built into neovim.

Really no idea. Try to see if you can find something with LspLog

**Reply by 0xccx** [reply]
Thanks, will take a look

`checkhealth` was showing some warnings that disappeared after installing some `libluajit-5.1` packages

I'm running Debian 11 with NVIM v0.9.0 compiled from source

**Comment by 0xccx** [comment, marked as answer]
Issue disappeared after upgrading from Debian 11 to Debian 12
